src/plugins/datepicker.ts

```diff
@@ -0,0 +1,60 @@
+import {Plugin, Cordova} from './plugin';
+
+/**
+ * The DatePicker plugin allows the user to fetch date or time using native dialogs.
+ *
+ * Platforms supported: iOS, Android, Windows
+ *
+ * Requires cordova-plugin-datepicker by VitaliiBlagodir that can be [found here](https://github.com/VitaliiBlagodir/cordova-plugin-datepicker).
+ *
+ * Install the plugin by running the following command:
+ * ```shell
+ * ionic plugin add cordova-plugin-datepicker
+ * ```
+ *
+ * @usage
+ * ```js
+ * DatePicker.
+ * ```
+ *
+ */
+@Plugin({
+  name: 'DatePicker',
+  plugin: '',
```

Comment:
I believe `plugin` is 'cordova-plugin-datepicker' and `pluginRef` is 'plugins.datePicker'